src/elements/post-card/post-card.composition.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import { PostCard, PostCardProps } from './post-card'
+
+export const HeroPostCard = (props: Partial<PostCardProps>) => (
+  <PostCard
+    src="https://www.jesusfilm.org/content/dam/jesusfilm/homepage/hero/hero-omn21.png"
+    title="His Shoes Led to Learning About Christianity"
+    excerpt="The call to prayer eminated from Omar's Lips - an eerie sound, out of place in the park. Omar and a group of 15 men bowed in sync."
+    category="Following Jesus"
+    onClick={(props.variant === 'hero' && props.onClick) || (() => alert('clicked'))}
+    variant="hero"
+    buttonText="Read Everything"
+  />
```

Comment:
If someone added props to `<HeroPostCard>` they should override the ones you've set here by default. This should be the case for all the following methods.

```suggestion
    {...props}
  />
```